src/components/SingleDatePicker.jsx

```diff
@@ -175,7 +177,8 @@ class SingleDatePicker extends React.PureComponent {
 
   /* istanbul ignore next */
   componentWillUnmount() {
-    if (this.removeEventListener) this.removeEventListener();
+    if (this.removeResizeEventListener) this.removeResizeEventListener();
```

Comment:
Renamed this to `removeResizeEventListener` because we've now added another event listener and it could lead to confusion otherwise 😺